Gopkg.toml

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
you can leave this file empty if you will not gonna force any specific version of anything right now...

Reply:
done